molecule/broker-scale-up/converge.yml

```diff
@@ -3,6 +3,9 @@
   hosts: kafka-broker3
   gather_facts: false
   tasks:
+    - import_role:
```

Comment:
Can we skip importing all variables and just check if kafka_controler group exists?

Reply:
in that case, we'll need to use the entire condition 'kafka_controller' in groups.keys() and groups['kafka_controller'] | length > 0 instead of kraft_enabled